model_compiler/model_compiler.py

```diff
@@ -25,9 +25,9 @@ def download_model(model, model_zoo_folder):
     downloader_cmd = [sys.executable, f"{model_downloader_path}"]
     downloader_cmd = np.concatenate((downloader_cmd, model_downloader_options))
     # print(downloader_cmd)
-    result = subprocess.run(downloader_cmd)
+    result = subprocess.run(downloader_cmd, capture_output=True)
```

Comment:
This makes model downloader/compiler fail on python3.6 (or older), 3.6 is the default on Ubuntu18.04. The `capture_output` option doesn't exist there:
`TypeError: __init__() got an unexpected keyword argument 'capture_output'`

Reply:
I did notice this bug introduced in this PR (so did sachin) and reverted it.
https://github.com/luxonis/depthai/pull/270#issuecomment-722628017
I will open a PR when refactoring is ready this weekend. 
Until then I will push a hotfix.

Reply:
Ah, ok, I missed seeing that PR. It may be fine then to wait until the refactoring is done.

Reply:
Hotfix pushed in #271